config/stp.py

```diff
@@ -500,6 +814,7 @@ def stp_vlan_enable(_db, vid):
             db.mod_entry('STP_VLAN_PORT', vlan_intf_key, vlan_intf_entry)
 
 
+# config spanning_tree vlan disable <vlan-id>
```

Comment:
This command is valid only for PVST. Please add check.

Reply:
I have addressed this comment, please review